### ISSUE TITLE
bug MongoDateValidator

### ISSUE THREAD

**Issue by reza-id** [yiisoft/yii2-mongodb#176]
### I'm trying to use validation rule to convert date to mongodb format
```php
 class Customer extends yii\mongodb\ActiveRecord
 {
     ...
     public function rules()
     {
         return [
             ['date', 'yii\mongodb\validators\MongoDateValidator', 'format' => 'php:Y-m-d', 'mongoDateAttribute' => 'dob']
         ];
     }
 }
 ```

### Additional info

| Q                      | A
| ---------------------- | ---
| Yii version            | 2.0.10
| Yii MongoDB version    | 2.1.2.0
| MongoDB server version |  3..4.1
| PHP version            |  7.0.8
| Operating system       | ubuntu0 16.04.3

**Comment by reza-id** [comment]
I think it because `MongoDB\BSON\UTCDateTime` should accept miliseconds as its parameter, but the $timestamp does not in miliseconds

**Comment by klimov-paul** [comment]
Unable to reproduce.
Described usecase is covered by unit test: 
https://github.com/yiisoft/yii2-mongodb/blob/master/tests/validators/MongoDateValidatorTest.php#L36-L44

**Comment by reza-id** [comment]
@klimov-paul have you looked at the value in db?
or try `$this->assertEquals('2015-05-08', $model->mongoDate->toDateTime()->format('Y-m-d'));` ?

**Comment by klimov-paul** [comment]
Resolved by commit 1c47907f7a8b865a7027ec2f8ee46468c7b271ad
